sdks/python/apache_beam/options/pipeline_options_validator_test.py

```diff
@@ -38,9 +38,12 @@
 # Mock runners to use for validations.
 class MockRunners(object):
   class DataflowRunner(object):
-    pass
+    def get_default_gcp_region(self):
```

Comment:
Clever, but a bit dangreous.

Reply:
We could also just add `--region` to all the tests if you would prefer.

Reply:
Not really. I like your solution better.